simd_emulation/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
I think the intention is to build this with make, so remove the Cargo.toml file.

Reply:
I'd like to be able to share the code with miri - and they want a Cargo file.
So keeping both.

(I may end up abandoning the Makefile at some point. It only works well as long as the entire crate is one file. But it makes it easy to find the bitcode file reliably afterwards so I like it for now.)